react-front-end/tsrc/legacycontent/LegacyContent.tsx

```diff
@@ -244,6 +246,27 @@ export const LegacyContent = React.memo(function LegacyContent({
     }
   }
 
+  /**
+   * temp.hn and temp.hb are two possible params contains in the request and response
+   * and are used in old UI to hide navigation (menu bar) and banner.
+   * In new UI, for normal legacy content we have a template props `fullscreenMode`
+   * received from server to hide menu bar, but not the error page.
+   * Thus, it is used to set `fullscreenMode` for error page.
+   */
+  function preUpdateFullscreenMode(submitValues: StateData) {
+    pipe(
```

Comment:
Can be nice and simple like:

```suggestion
  const preUpdateFullscreenMode = (submitValues: StateData) =>
    pipe(
```